### PR TITLE
Add scaffold plugin --activate-network option

### PR DESCRIPTION
**Pull request by acusti**
Includes new functional test for `wp scaffold _s --activate`; I’m planning on also creating a parallel issue for that command, though I’m not sure what the option should be called, because the nomenclature seems different (`wp theme enable` vs `wp plugin activate`).

See #1709

### CODE REVIEW

File: features/scaffold.feature [hunk 1]
```diff
@@ -1,10 +1,8 @@
 Feature: WordPress code scaffolding
 
-  Background:
-    Given a WP install
-
```
Comment (danielbachhuber) [inline features/scaffold.feature]: Why did you change this?

File: php/commands/scaffold.php [hunk 2]
```diff
@@ -447,6 +450,10 @@ function plugin( $args, $assoc_args ) {
 		if ( isset( $assoc_args['activate'] ) ) {
 			WP_CLI::run_command( array( 'plugin', 'activate', $plugin_slug ) );
 		}
+
+		if ( isset( $assoc_args['activate-network'] ) ) {
```
Comment (danielbachhuber) [inline php/commands/scaffold.php]: This should probably be `elseif`

**Comment by danielbachhuber** [comment]
Couple small nits, otherwise this looks good.

File: features/scaffold.feature (hunk 1 above)
Comment (acusti) [inline features/scaffold.feature]: I needed to use `Given a WP multisite install` on the `--activate-network` test

**Comment by acusti** [comment]
Right on. Should I create an issue for `wp scaffold _s` and make the same change there? I noticed the API for `wp theme` and `wp plugin` diverge (`wp theme enable` vs `wp plugin activate`).

**Comment by danielbachhuber** [comment]
> Should I create an issue for wp scaffold _s and make the same change there? I noticed the API for wp theme and wp plugin diverge (wp theme enable vs wp plugin activate).

Regarding?

**Comment by acusti** [comment]
Regarding adding an option for `--activate-network`, or whatever the nomenclature should be, like: `wp scaffold _s starter-theme --activate-network`

**Comment by danielbachhuber** [comment]
> Regarding adding an option for --activate-network, or whatever the nomenclature should be, like: wp scaffold _s starter-theme --activate-network

Oh. We can discuss in a new issue. I think the more appropriate syntax would be `--enable-network`

**Comment by acusti** [comment]
Alright, sounds good. Did you see my reply about removing the background section from the functional tests in `scaffold.feature`?

**Comment by danielbachhuber** [comment]
> Did you see my reply about removing the background section from the functional tests in scaffold.feature?

Yep, that's fine.

File: php/commands/scaffold.php [hunk 3]
```diff
@@ -446,6 +455,8 @@ function plugin( $args, $assoc_args ) {
 
 		if ( isset( $assoc_args['activate'] ) ) {
 			WP_CLI::run_command( array( 'plugin', 'activate', $plugin_slug ) );
+		} else if ( isset( $assoc_args['activate-network'] ) ) {
+			WP_CLI::run_command( array( 'plugin', 'activate', $plugin_slug), array( 'network' => true ) );
```
Comment (danielbachhuber) [inline php/commands/scaffold.php]: Why is this code in this PR?

Comment (acusti) [inline php/commands/scaffold.php]: That’s the same line you commented on earlier that you suggested I change to an `else if`, no? This line fixes #1709, while the other lines fix #1711.

Comment (danielbachhuber) [inline php/commands/scaffold.php]: Changes to plugin activation should be in a separate pull request from those to theme activation.

Comment (acusti) [inline php/commands/scaffold.php]: Okay. I rolled back to the last commit before I added the multisite-related changes to theme activation.

**Comment by danielbachhuber** [comment]
@acusti Thanks for your work on this!

**Comment by acusti** [comment]
@danielbachhuber Thanks for merging it in!
